docs/reference/action_list.rst

```diff
@@ -147,75 +147,97 @@ Available types and associated options
++                                  +---------------------+-----------------------------------------------------------------------+
+|                                  | timezone            | Second argument for Twig's ``date`` function                          |
++                                  +---------------------+-----------------------------------------------------------------------+
+|                                  | editable            | Yes/No; editable allows to edit directly from the list if authorized. |
++----------------------------------+---------------------+-----------------------------------------------------------------------+
+| TemplateRegistry::TYPE_TIME      | format              | A format understandable by Twig's ``date`` function.                  |
++                                  +---------------------+-----------------------------------------------------------------------+
+|                                  | timezone            | Second argument for Twig's ``date`` function                          |
++----------------------------------+---------------------+-----------------------------------------------------------------------+
+| TemplateRegistry::TYPE_DATETIME  | format              | A format understandable by Twig's ``date`` function.                  |
++                                  +---------------------+-----------------------------------------------------------------------+
+|                                  | timezone            | Second argument for Twig's ``date`` function                          |
++----------------------------------+---------------------+-----------------------------------------------------------------------+
+| TemplateRegistry::TYPE_EMAIL     | as_string           | Renders the email as string, without any link.                        |
++                                  +---------------------+-----------------------------------------------------------------------+
+|                                  | subject             | Add subject parameter to email link.                                  |
++                                  +---------------------+-----------------------------------------------------------------------+
+|                                  | body                | Add body parameter to email link.                                     |
++                                  +---------------------+-----------------------------------------------------------------------+
+|                                  | editable            | Yes/No; editable allows to edit directly from the list if authorized. |
++----------------------------------+---------------------+-----------------------------------------------------------------------+
+| TemplateRegistry::TYPE_PERCENT   |                     | Renders value as a percentage.                                        |
```

Comment:
It's weird to see that some types have a global description while others have only a description for their options.